pom.xml

```diff
@@ -14,9 +14,9 @@
 
   <properties>
     <stack.version>4.4.0-SNAPSHOT</stack.version>
-    <curator.version>4.3.0</curator.version>
-    <zookeeper.version>3.5.9</zookeeper.version>
-    <junit.version>4.13.1</junit.version>
+    <curator.version>5.4.0</curator.version>
+    <zookeeper.version>3.7.1</zookeeper.version>
+    <junit.version>5.6.2</junit.version>
```

Comment:
We'll stick to junit4 as the rest of the vert.x stack

Reply:
Hi,

I reverted junit to 4.13.1
As curator-test was bringing junit-jupiter-api (junit 5) in the classpath, I had to exclude the depdency from curator-test in order to still have the test working properly.